cocos/3d/skeletal-animation/skeletal-animation-state.ts

```diff
@@ -159,7 +159,7 @@ export class SkeletalAnimationState extends AnimationState {
                     mat = new Mat4();
                 }
                 const tfm = { pos: new Vec3(), rot: new Quat(), scale: new Vec3() };
-                Mat4.toRTS(mat, tfm.rot, tfm.pos, tfm.scale);
+                Mat4.toSRT(mat, tfm.rot, tfm.pos, tfm.scale);
```

Comment:
Why change it?

Reply:
![WeChatfe1cc2096000cbab9d0450154501527e](https://github.com/user-attachments/assets/48287d41-cb56-48b6-ad3c-05df4aa6a99c)